src/server/worker/driver/driver.go

```diff
@@ -525,6 +535,7 @@ func (d *driver) UserCodeEnv(
 	outputCommit *pfs.Commit,
 	inputs []*common.Input,
 	authToken string,
```

Comment:
I think authToken & pachToken here is redundant. We should only use the Job's Auth Token now in the UserCodeEnv instead of both the Job's and the Pipeline's Token